nms/app/packages/magmalte/app/components/GatewayUtils.js

```diff
@@ -177,6 +177,10 @@ type GatewayStatusPayload = {
 
 const GATEWAY_KEEPALIVE_TIMEOUT_MS = 1000 * 5 * 60;
 
+export const GOOD_GATEWAY = 'Good';
+
+export const BAD_GATEWAY = 'Bad';
```

Comment:
Lol rename these constants to indicate it's for labeling a healthy or unhealthy gateway

Reply:
Haha, my bad. I just did that.